lib/modules/manager/cnb/extract.ts

```diff
@@ -0,0 +1,108 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import { getDep } from '../dockerfile/extract';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { type ProjectDescriptor, ProjectDescriptorToml } from './schema';
+
+const dockerPrefix = regEx(/^docker:\/?\//);
+const buildpackRegistryRef = regEx(/^[a-z0-9\-\.]+\/[a-z0-9\-\.]+(?:@.+)?$/);
```

Comment:
ESLint is complaining about this line:

> Unnecessary escape character: `\..`

I don't see a _literal_ match for `\..` on this line though.

Reply:
fixed